theme/static/js/custom.js

```diff
@@ -191,7 +176,185 @@ $(document).ready(function () {
             }
         },
         error: showUniversalMessage()
-    })
+    });
+
+    // Event trigger for profile preview
+    $(".profile-preview").click(function () {
+        // Move the profile card below the clicked item
+        var profileCard = $(this).parent().find(".profile-card");
+        profileCard.css("top", ($(this).position().top + 30) + "px");
```

Comment:
How about the view on mobile devices?

Reply:
I tested this on mobile devices and it worked well too. I will give it another round of testing just to be sure.